extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -507,31 +534,21 @@ export class AmpLightboxGallery extends AMP.BaseElement {
     }
     if (this.controlsMode_ == LightboxControlsModes.CONTROLS_HIDDEN) {
       this.showControls_();
-    } else {
+    } else if (!this.container_.hasAttribute('gallery-view')) {
       this.hideControls_();
-      // Toggle version of hide uses fade animation effects
-      this.topBar_.classList.add('fade-out');
-      this.descriptionBox_.classList.add('fade-out');
-    } }
+    }
+  }
 
   /**
    * Show lightbox controls.
    * @private
    */
   showControls_() {
-    this.carousel_.setAttribute('controls', '');
-
-    this.topBar_.classList.remove('fade-out');
-    this.topBar_.classList.remove('hidden');
-    this.topBar_.classList.add('show');
-    this.topBar_.classList.add('fade-in');
-
-    this.descriptionBox_.classList.remove('fade-out');
+    this.controlsContainer_.classList.remove('fade-out');
+    this.controlsContainer_.classList.remove('hidden');
```

Comment:
nit: I would prefix these classnames with `i-amphtml-` so they are less likely to conflict with classnames from the publisher. I imagine this would be an issue with automatically converted documents. I know you didn't add these classnames in this PR but it would be a good change to write CSS defensively. Feel free to do it in a separate PR, or not at all